exporters/otlp/otlplog/otlploggrpc/config.go

```diff
@@ -360,8 +360,9 @@ func WithTimeout(duration time.Duration) Option {
 // explicitly returns a backoff time in the response, that time will take
 // precedence over these settings.
 //
-// These settings do not define any network retry strategy. That is entirely
-// handled by the gRPC ClientConn.
+// These settings define the retry strategy implemented by the exporter.
+// These settings do not define any network retry strategy
+// that is handled by the gRPC ClientConn.
```

Comment:
Should there be a new phrase between those two lines? (same for the two other of course)

```suggestion
// These settings do not define any network retry strategy.
// That is handled by the gRPC ClientConn.
```

Reply:
I personally prefer keep `that` connected to the previous word so it is clear that `network retry strategy` is handled by gRPC ClientConn and we are not using it for retry.

Reply:
Breaking these into two sentences makes sense to me as well. They are two distinct, related ideas.

Using a single sentence reads in a way that conveys the settings are not reaching into the gRPC ClientConn and changing a network retry strategy. But the intent here is to state that the settings do not define a network retry strategy. Rather, it is the gRPC ClientConn that does that. That is communicated by breaking these into two sentences.

Reply:
I see. It makes sense.